src/packagedcode/buildpack.py

```diff
@@ -0,0 +1,143 @@
+#
+# Copyright (c) nexB Inc. and others. All rights reserved.
+# ScanCode is a trademark of nexB Inc.
+# SPDX-License-Identifier: Apache-2.0
+# See http://www.apache.org/licenses/LICENSE-2.0 for the license text.
+# See https://github.com/nexB/scancode-toolkit for support or download.
+# See https://aboutcode.org for more information about nexB OSS projects.
+#
+
+import toml
+from packagedcode import models
+from packageurl import PackageURL
+
+class BuildpackHandler(models.DatafileHandler):
```

Comment:
This should be a subclass of models.NonAssemblableDatafileHandler since we want to detect this as package data, but do not want to create top level packages/assign resources to this package data.